.github/workflows/deploy_staging.yml

```diff
@@ -0,0 +1,29 @@
+name: Deploy to staging
+
+concurrency:
+  group: ${{ github.ref }}
+  cancel-in-progress: true
+
+on:
+  push:
+    branches:
+      - staging
+
+jobs:
+  deploy:
+    name: Deploy
+    uses: ./.github/workflows/aws_deploy.yml
+    with:
+      aws-region: "ap-southeast-1"
+      cicd-role: "arn:aws:iam::646878943557:role/redirection-github-oidc-role"
```

Comment:
should we var this?

Reply:
the `cicd-role` is a var that's used inside the `aws_deploy.yml`